libs/entitlement/src/test/java/org/elasticsearch/entitlement/runtime/policy/PolicyManagerTests.java

```diff
@@ -59,6 +59,7 @@ public void testGetEntitlementsThrowsOnMissingPluginUnnamedModule() {
             List.of(),
             Map.of("plugin1", createPluginPolicy("plugin.module")),
             c -> "plugin1",
+            NO_ENTITLEMENTS_MODULE,
```

Comment:
Rather than reuse NO_ENTITLEMENTS_MODULE, can we create a test specific agents module?

Reply:
Changed.